devices/tuya/_TZE200_TS0601_water_valve.json

```diff
@@ -0,0 +1,195 @@
+{
+	"schema": "devcap1.schema.json",
+	"manufacturername": ["_TZE200_a7sghmms", "_TZE200_sh1btabb"],
+	"modelid": ["TS0601", "TS0601"],
+	"vendor": "GIEXperience",
+	"product": "Smart water valve",
+	"sleeper": false,
+	"status": "Gold",
+	"subdevices": [{
+			"type": "$TYPE_ON_OFF_OUTPUT",
+			"restapi": "/lights",
+			"uuid": [
+				"$address.ext",
+				"0x01"
+			],
+			"items": [{
+					"name": "attr/id"
+				},
+				{
+					"name": "attr/lastannounced"
+				},
+				{
+					"name": "attr/lastseen"
+				},
+				{
+					"name": "attr/manufacturername"
+				},
+				{
+					"name": "attr/modelid"
+				},
+				{
+					"name": "attr/name"
+				},
+				{
+					"name": "attr/swversion",
+					"parse": {"fn": "zcl:attr", "ep": 1, "cl": "0x0000", "at": "0x0001", "script": "tuya_swversion.js"},
+				        "read": {"fn": "zcl:attr", "ep": 1, "cl": "0x0000", "at": "0x0001"}
+				},
+				{
+					"name": "attr/type"
+				},
+				{
+					"name": "attr/uniqueid"
+				},
+				{
+					"name": "state/on",
+					"refresh.interval": 300,
+					"read": {
+						"fn": "tuya"
+					},
+					"write": {
+						"dpid": 2,
+						"dt": "0x10",
+						"eval": "Item.val == 1 ? 1 : 0;",
+						"fn": "tuya"
+					},
+					"parse": {
+						"dpid": 2,
+						"eval": "Item.val = Attr.val;",
+						"fn": "tuya"
+					},
+					"default": 0
+				},
+				{
+					"name": "config/tuya_unlock"
+				},
+				{
+					"name": "state/reachable"
+				}
+			]
+		},
+		{
+			"type": "$TYPE_CONSUMPTION_SENSOR",
+			"restapi": "/sensors",
+			"uuid": [
+				"$address.ext",
+				"0x01",
+				"0x0702"
+			],
+			"items": [{
+					"name": "attr/id"
+				},
+				{
+					"name": "attr/lastannounced"
+				},
+				{
+					"name": "attr/lastseen"
+				},
+				{
+					"name": "attr/manufacturername"
+				},
+				{
+					"name": "attr/modelid"
+				},
+				{
+					"name": "attr/name"
+				},
+				{
+					"name": "attr/swversion",
+					"parse": {"fn": "zcl:attr", "ep": 1, "cl": "0x0000", "at": "0x0001", "script": "tuya_swversion.js"},
+				        "read": {"fn": "zcl:attr", "ep": 1, "cl": "0x0000", "at": "0x0001"}
+				},
+				{
+					"name": "attr/type"
+				},
+				{
+					"name": "attr/uniqueid"
+				},
+				{
+					"name": "config/battery",
+					"parse": {
+						"dpid": 108,
+						"eval": "Item.val = Attr.val;",
+						"fn": "tuya"
+					},
+					"default": 0
+				},
+				{
+					"name": "config/duration",

```

Comment:
I don't think we have anything for last irrigation duration.  I don't think it should be on the `sensors` resource, but on the `lights` resource instead, where you handle the on/off state.  Also, it should be a `state` item, as it's not configurable.

Reply:
The DDF was created through @BabaIsYou advice (see [#6944](https://github.com/dresden-elektronik/deconz-rest-plugin/issues/6944)). I was able to make the device working (on/off) and have added also some printscreen of eve app on the issue request.

Reply:
I see I'm already quoted on that issue.

To get remaining duration in Eve (and even in Apple's Home app), expose the device as a _Valve_ accessory.  Homebridge deCONZ does expect `state/ontime` to support that, though.  Never tried to combine that with consumption, though.